cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.1 on 2024-10-31 06:38
+
+import cinema.models

```

Comment:
Ensure that the `movie_image_path` function is correctly defined in `cinema.models`. This function should handle the logic for determining the file path where the image will be stored.